tests/Doctrine/Tests/ORM/Functional/Ticket/GH5762Test.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @group GH-5762
+ */
+class GH5762Test extends \Doctrine\Tests\OrmFunctionalTestCase
+{
+
+    protected function setup()
+    {
+        parent::setup();
+
+        $this->_schemaTool->createSchema(array(
+            $this->_em->getClassMetadata(__NAMESPACE__ . '\GH5762Driver'),
```

Comment:
Good point. Doctrine was stuck with 5.4 in my mind but looks like it's time to finally embrace the good things, isn't it?